core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -98,6 +98,27 @@ package object config {
       .booleanConf
       .createWithDefault(false)
 
+  private[spark] val EVENT_LOG_GARBAGE_COLLECTION_METRICS =
+    ConfigBuilder("spark.eventLog.logStageExecutorGCMetrics.enabled")
+      .booleanConf
+      .createWithDefault(false)
+
+  private[spark] val EVENT_LOG_ADDITIONAL_YOUNG_GENERATION_GARBAGE_COLLECTORS =
+    ConfigBuilder("spark.eventLog.additionalYoungGenerationGarbageCollectors")
+      .doc("Names of additional young generation garbage collector, " +
+        "usually is the return of GarbageCollectorMXBean.getName, e.g. ParNew.")
```

Comment:
Is it better to add default set of names? Users usually do not see source code.

Reply:
Correct. I improved.